nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/QueryElasticsearchHttp.java

```diff
@@ -175,16 +197,21 @@
             .allowableValues(TARGET_FLOW_FILE_CONTENT, TARGET_FLOW_FILE_ATTRIBUTES)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR).build();
 
-    private static final Set<Relationship> relationships;
+    public static final PropertyDescriptor ROUTING_QUERY_INFO_STRATEGY = new PropertyDescriptor.Builder()
+            .name("routing-query-info-strategy")
+            .displayName("Routing Strategy for Query Info")
+            .description("Specifies when to generate and route Query Info after a successful query")
+            .expressionLanguageSupported(false)
```

Comment:
As of PR #2205 I believe this has to be changed from false to ExpressionLanguageScope.NONE

Reply:
Fixed thanks